{"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError"]]},"metadata":{"roleHeading":"API Collection","title":"LocalizedError Implementations","role":"collectionGroup","modules":[{"name":"Auth0"}]},"topicSections":[{"title":"Instance Properties","generated":true,"identifiers":["doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/failureReason","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/helpAnchor","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/recoverySuggestion"]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/auth0\/authenticationerror\/localizederror-implementations"]}],"schemaVersion":{"major":0,"patch":0,"minor":3},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/LocalizedError-Implementations"},"kind":"article","sections":[],"references":{"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError":{"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError","title":"AuthenticationError","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AuthenticationError","kind":"identifier"}],"abstract":[{"type":"text","text":"Represents an error during a request to the Auth0 Authentication API."}],"url":"\/documentation\/auth0\/authenticationerror","role":"symbol","type":"topic","kind":"symbol","navigatorTitle":[{"text":"AuthenticationError","kind":"identifier"}]},"doc://com.auth0.Auth0/documentation/Auth0":{"type":"topic","kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","url":"\/documentation\/auth0","title":"Auth0","abstract":[{"type":"text","text":"SDK for Apple platforms."}],"role":"collection"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/failureReason":{"kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/failureReason","abstract":[],"title":"failureReason","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?"}],"url":"\/documentation\/auth0\/authenticationerror\/failurereason","type":"topic"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/helpAnchor":{"kind":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"helpAnchor"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"title":"helpAnchor","url":"\/documentation\/auth0\/authenticationerror\/helpanchor","type":"topic","abstract":[],"role":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/helpAnchor"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/recoverySuggestion":{"title":"recoverySuggestion","kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/recoverySuggestion","type":"topic","url":"\/documentation\/auth0\/authenticationerror\/recoverysuggestion","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"role":"symbol","abstract":[]}}}